tests/functional/api/BackbeatServer.js

```diff
@@ -267,7 +277,15 @@ describe('Backbeat Server', () => {
             statsClient.reportNewRequest(`${site2}:${BYTES_DONE}`, 1874);
             statsClient.reportNewRequest(`${site2}:${BYTES_FAIL}`, 575);
 
+            const testVersionId =
+                '3938353030303836313334343731393939393939524730303120203';
+            const members = [
+                `test-bucket:test-key:${testVersionId}0:${site1}`,
+                `test-bucket:test-key:${testVersionId}1:${site2}`,
+            ];
+
             return async.parallel([
+                next => addMembers(redisClient, site1, members, next),
```

Comment:
nice 👍 my fake data added below on L#297 and L#308 may create conflicts with tests possibly?

I basically put in 10 fake entries per hour for each `site1` and `site2`. I also did not form the score and member values as I did it solely for metrics. We can probably drop my changes and use your added members instead? Or I can form my score and members to be more realistic

Reply:
I added +2 to those value checks. Does that work?

Reply:
sounds good